kinbot/mess.py

```diff
@@ -196,6 +239,7 @@ def write_input(self, qc):
 
         # write the mess input for the different blocks
         for uq_iter in range(self.par['uq_n']):
+            header = self.write_header(uq_iter)
```

Comment:
moved create header into uq portion of code so that epsilon/sigma/energy relaxation factors can be edited for each iteration.